health_check/backends.py

```diff
@@ -11,6 +11,9 @@
 class BaseHealthCheckBackend:
     def __init__(self):
         self.errors = []
+        self.warnings = []
+        self.critical = getattr(self, 'critical', True)
+        self.description = getattr(self, 'description', '')
```

Comment:
Same here....

Reply:
Same as above. Just ensuring it's not None.